src/insert.rs

```diff
@@ -85,6 +108,13 @@ mod tests {
             }
         );
     }
+    #[test]
+    fn simple_insert_reformat() {
+        let qstring = "INSERT INTO users VALUES (42, \"test\");";
+        let expected = "INSERT INTO users (0, 1) VALUES (42, 'test')";
```

Comment:
Why is the expected output `(0, 1)` here? Shouldn't that be column names? Arguably they're not defined in the query, so it's hard to know without having a schema handy, but I wouldn't expect SQL to output column IDs here. Plus, [at least the SQLite spec](https://sqlite.org/lang_insert.html) allows the `... INTO users VALUES ...` syntax.

Reply:
I got confused with this code too https://github.com/ms705/nom-sql/blob/19c5ae7c0335fe6328d16281b0ea0fce5ea550e6/src/insert.rs#L52-L58 So, I just leave it as it was and make the test satisfy, instead of doing something here that may break what was the original purpose of this code.